modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/source/SourceConfigurer.java

```diff
@@ -126,39 +127,31 @@ public Object build(ValueResolvingContext context) throws MuleException {
 
         };
 
-    CoreEvent initialiserEvent = null;
-    ValueResolvingContext context = null;
-    try {
-      initialiserEvent = getNullEvent(muleContext);
-      context = ValueResolvingContext.builder(initialiserEvent, expressionManager).withConfig(config).build();
-      Object configuredSource = builder.build(context);
+    return withNullEvent(event -> {
+      try (ValueResolvingContext context = ValueResolvingContext.builder(event, expressionManager).withConfig(config).build()) {
+        Object configuredSource = builder.build(context);
 
-      Source sdkSource = SdkSourceAdapterFactory.createAdapter(configuredSource);
+        Source sdkSource = SdkSourceAdapterFactory.createAdapter(configuredSource);
 
-      if (sdkSource instanceof PollingSource) {
-        ValueResolver<?> valueResolver = resolverSet.getResolvers().get(SCHEDULING_STRATEGY_PARAMETER_NAME);
-        if (valueResolver != null) {
-          context = ValueResolvingContext.builder(initialiserEvent, expressionManager).build();
-          SchedulingStrategy scheduler = (SchedulingStrategy) valueResolver.resolve(context);
+        if (sdkSource instanceof PollingSource) {
+          SchedulingStrategy scheduler;
+          ValueResolver<?> valueResolver = resolverSet.getResolvers().get(SCHEDULING_STRATEGY_PARAMETER_NAME);
+          if (valueResolver == null) {
+            scheduler = new FixedFrequencyScheduler(60000, 0, MILLISECONDS);
```

Comment:
What about putting this default in the SourceParameterizer, if it is a polling source and no scheduling strategy is given?

Putting it here makes it a hidden default that may cover an issue in future sources.

Reply:
I tried originally but introduces a lot of other issues. This doesn't hide a default, since the default is already present on the XSD and the EM. Quite the contrary, this fixes a bug in the original code in which if a polling source is not provided with a scheduler, then it never starts and dies silently